packages/gatsby-theme-newrelic/src/components/CookieConsentDialog.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'js-cookie';
+import { css } from '@emotion/core';
+import ExternalLink from './ExternalLink';
+import Button from './Button';
+
+const cookieName = 'newrelic-gdpr-consent';
+
+const CookieConsentDialog = () => {
+  const [isCookieSet, setIsCookieSet] = useState(Cookies.get(cookieName));
+
+  const writeCookie = (answer) => {
+    const currentEnvironment =
+      process.env.ENV || process.env.NODE_ENV || 'development';
+    const options = { expires: 365 /* days */ };
+    if (currentEnvironment !== 'development') {
+      options.domain = 'newrelic.com';
+    }
+
+    Cookies.set(cookieName, String(!!answer), options);
```

Comment:
[nit] It looks like the `answer` passed here is always a boolean, so no need for the `!!`. You can just `String(answer)`